apps/meeting/endpoints/v1/email.py

```diff
@@ -0,0 +1,144 @@
+    MAIL_TLS=True,
+    MAIL_SSL=False,
+    USE_CREDENTIALS=True,
+)
+
+
+def get_matching_failure_text():
+    """
+    Returns failing text of email
+    """
+    return """
+Hello,
+this is GaussTalk. 
+
+We are sorry to inform you that the 1:1 matching you have requested have failed due to insufficient participants.
+If you would like to do random 1:1s tomorrow please register by our website.
+We will match you as soon as possible.
+
+Sorry for the inconvenience.
+
+GaussTalk 
+    """
+
+
+def get_matching_success_text(recipient, matched_recipient_list):
+    """
+    Returns success text of email
+    """
+    matched_opponents = None
+    for group in matched_recipient_list:
+        if recipient in group:
+            others = [x for x in group if x != recipient]
+            matched_opponents = others
+
+    opponent_string = ""
+    if len(matched_opponents) > 1:
+        opponent_string = matched_opponents[0] + " and " + matched_opponents[1]
+    else:
+        opponent_string = matched_opponents[0]
+    return """
+Hello,
+this is GaussTalk. 
+Below includes the email of your random 1:1.
+Please contact your match to set up a time for your 1:1!
+
+You (""" + recipient + """) have been matched with """ + opponent_string + """.
+
+Enjoy your random 1:1!
+GaussTalk
+"""
+
+
+@router.post("/")
+async def simple_send(
+        background_tasks: BackgroundTasks,
+        database=Depends(get_mongo)
+):
+    """Get yesterday's email and send it"""
+    yesterday = datetime.date.today() - datetime.timedelta(days=1)
+    # today = datetime.date.today()
+    try:
+        emails = (database.meetings.find({'date': yesterday.strftime('%Y-%m-%d')},
+                                         {'_id': 0, 'date': 0}))
+        recipients = []
+        for email in emails:
+            recipients.append(email['mail'])
+    except (TypeError, KeyError):
+        return JSONResponse(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                            content={"message": "email sending failed"})
+
+    def match_recipients(people):
+        random.shuffle(people)
+        list_of_groups = [people[i:i + 2] for i in range(0, len(people), 2)]
+        if len(list_of_groups[-1]) == 1:
+            solo = list_of_groups.pop()
+            list_of_groups[-1].append(*solo)
+        return list_of_groups
+
+    fastmail = FastMail(conf)
+
+    if len(recipients) == 0:
+        return JSONResponse(status_code=200, content={"message": "no emails to send"})
+
+    if len(recipients) == 1:
+        text = get_matching_failure_text()
+        message = MessageSchema(
+            subject="1:1 Matching Complete",
+            recipients=recipients,  # List of recipients, as many as you can pass
+            body=text,
+            subtype="plain"
+        )
+        background_tasks.add_task(fastmail.send_message, message)
+    else:
+        matched_recipients = match_recipients(recipients)
+        for recipient in recipients:
```

Comment:
This algorithm is not optimized. Sending to all recipients is O(n^2). But, it can be improved to O(n). But it looks working well, let's fix it in a next PR.

```suggestion
        for group in matched_recipients:
            if len(group) == 1:
                // sorry
            elif len(group) == 2:
               cheolsoo, younghee = group
                // send 1:1 mail each other
            elif len(group) == 3:
                cheolsoo, younghee, yooha = group
                // send 1:1:1 mail each other
```